Week3/More JavaScript.js

```diff
@@ -79,15 +101,19 @@ console.log(listOfVehicle[2]);
 // 9. Change the function vehicle to use the list of question 7. So that vehicle("green", 3, 1) prints "a green new bike".
 
 function newVehicle(color, category, age) {
-    if (age >= 4) {
-        age = "used";
+  if (category > 5) {
```

Comment:
Here you could also use `listOfVehicle.length`. You can read more about it here: https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Array

Reply:
👍